src/overlays/actors/ovl_En_Dekunuts/z_en_dekunuts.h

```diff
@@ -6,9 +6,20 @@
 
 struct EnDekunuts;
 
+typedef void (*EnDekunutsActionFunc)(struct EnDekunuts*, GlobalContext*);
+
 typedef struct EnDekunuts {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1C8];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ EnDekunutsActionFunc actionFunc;
+    /* 0x0194 */ u8 playWalkSound;
+    /* 0x0195 */ u8 runAwayCount;
+    /* 0x0196 */ s16 animFlagAndTimer;
+    /* 0x0198 */ s16 runDirection;
+    /* 0x019A */ volatile s16 unk_19A;
```

Comment:
keyword volatile like this is nowhere else to be seen. So it has to be super fake, right? This was the only way I was able to make EnDekunuts_Init match, but I would gladly try any ideas